src/css/skins-test.less

```diff
@@ -1,25 +1,31 @@
-@import "imports/vars";
```

Comment:
Can we remove this file? You left duplicate imports of "jwplayerelement" down there...